lib/screens/create_game2/create_game_map.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:niira/loading.dart';
+import 'package:niira/services/location_service.dart';
+import 'package:niira/utilities/map_styles/create_game_map.dart';
+import 'package:provider/provider.dart';
+
+class CreateGameMap extends StatefulWidget {
+  @override
+  State<CreateGameMap> createState() => CreateGameMapState();
+}
+
+class CreateGameMapState extends State<CreateGameMap> {
+  final Completer<GoogleMapController> _controller = Completer();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: FutureBuilder(
+        future: context.watch<LocationService>().getUsersCurrentLocation(),
+        builder: (context, snapshot) {
+          if (snapshot.hasData) {
+            /// Put current users location in `LatLng` object so we can use it
+            /// in flutter_google_maps objects
+            final _playerLocation = LatLng(
```

Comment:
you could use a toLatLng() function to Location which would make this a bit cleaner imo

Reply:
so it would then just be snapshot.data.toLatLng()

Reply:
woop woop beautiful